app/views/steps/completion/confirmation/show.en.html.erb

```diff
@@ -1,14 +1,22 @@
 <% title 'Application submitted' %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <div class="grid_content_header govuk-box-highlight">
-      <h1 class="app__heading heading-xlarge gv-u-heading-xxlarge">Your application has been submitted</h1>
-      <p class="display-block util_mt-none util_mb-none">Your reference code is:</p>
-      <h3 class="heading-medium util_mt-none util_mb-medium"><%= current_c100_application.reference_code %></h3>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-panel govuk-panel--confirmation">
+      <h1 class="govuk-panel__title">
+        Your application
+        <br>
```

Comment:
Let's to get rid of this `<br>` as they don't work nice with different screen sizes and I also believe they are not recommended for screen readers.